src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Controllers/Api/Public.Data/PreviewTokenControllerTests.cs

```diff
@@ -309,8 +309,9 @@ await TestApp.AddTestData<PublicDataDbContext>(context =>
                 Label = previewToken.Label,
                 Status = PreviewTokenStatus.Active,
                 Created = previewToken.Created,
+                Activates = previewToken.Created,
```

Comment:
Highlighting a change to `Activates` which has crept into this PR. Not sure if this is intentional or not.